website/src/views/buckets.js

```diff
@@ -31,17 +27,6 @@ export default function Buckets(){
             console.log("test")
```

Comment:
```suggestion
```
Totally not for ur code, but I just noticed this so might as well include it.

Reply:
removed extra console.log line